integration/test/performancetest/resources/config10.json

```diff
@@ -0,0 +1,162 @@
+{
+ "agent": {
+  "metrics_collection_interval": 1,
```

Comment:
Same question as before: I see the value of using collecting high-resolution metrics for the metrics_collected but I'm not sure if the trade off is worth it? 10% different in values and higher cost (with a 60x collection rate). Therefore, I propose we should keep the 60 and also I am not sure why we would change the metrics_collection_interval for agent at an interval of 1

Reply:
Metric collection does not have a high impact on agent resource usage, so I figured that increasing metric collection interval and adding more metrics collected would make for a better benchmark of the agent's resources and cover more use cases for the average customer.

Reply:
Yes it does. More collected metrics, more metrics need to aggregate even though the batch of MetricDatum is the same. Therefore, it could cause a spike in the moment of collection. In exchange between more noise in the performance metric and  10% different in value, lower cost, I am stand by 60 though.

Reply:
![Screen Shot 2022-07-18 at 10 16 40 AM](https://user-images.githubusercontent.com/107051245/179531806-7e029441-1549-4999-804a-c622db881fce.png)
I tested this on an ec2 instance, the difference between a 1s collection interval on all metrics and a 60s collection interval on all metrics was ~2% cpu on a t2.micro instance